client/src/actions/session-actions.js

```diff
@@ -54,11 +54,11 @@ export default {
                 data: {
                     userId: rememberData.userId,
                     rememberToken: rememberData.token,
+                    staff: (rememberData.isStaff === "true") ? 1 : 0,
```

Comment:
Is there any reason not to store 1 and 0 (as in other local storage data) instead of the string "true" / "false"?